src/rogue/utilities/Prbs.cpp

```diff
@@ -170,13 +170,25 @@ void ru::Prbs::sendCount(bool state) {
    sendCount_ = state;
 }
 
-void ru::Prbs::flfsr(ru::PrbsData & data) {
-   bool bit = false;
-
-   for (uint32_t x=0; x < tapCnt_; x++) bit ^= data[taps_[x]];
+void ru::Prbs::flfsr(uint8_t * data) {
+   uint32_t iByte;
+   uint32_t iBit;
+   uint32_t x;
+   bool msbOut;
+   bool lsbIn = false;
+
+   for (x=0; x < tapCnt_; x++) {
+      iByte = taps_[x] / 8;
```

Comment:
instead of `iByte = taps_[x] / 8;`, wouldn't a `iByte = taps_[x] >> 3;` run faster?  That's my experience with microcontrollers but a modern CPU probably does it in a single cycle